From 75435d0906fa6722155a445e694b5ff45f71608a Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Thu, 25 Feb 2010 10:10:14 +1000
Subject: [PATCH] Add Euro and New Shekel sign to israeli layout.

According to [1], the standard layout requires the shekel sign on AltGr + a,
although Windows has it on AltGr + 4. The latter is already in use for some
other character, so let's put it where the standard requires it to be.
Euro sign goes on AltGr + e.

This requires the inclusion of level3(ralt_switch) to get the IsoLevel3.

[1] http://en.wikipedia.org/wiki/Shekel_sign

Fedora Bug https://bugzilla.redhat.com/show_bug.cgi?id=542879

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
---
 symbols/il |    6 ++++--
 1 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/symbols/il b/symbols/il
index 4a3055d..cc29b12 100644
--- a/symbols/il
+++ b/symbols/il
@@ -16,7 +16,6 @@ xkb_symbols "basic" {
 
     key <AD01> { [ slash,	Q		]	};
     key <AD02> { [ apostrophe,	W		]	};
-    key <AD03> { [ hebrew_qoph,	E		]       };
     key <AD04> { [ hebrew_resh,	R		]	};
     key <AD05> { [ hebrew_aleph,T		]       };
     key <AD06> { [ hebrew_tet,	Y		]       };
@@ -25,7 +24,6 @@ xkb_symbols "basic" {
     key <AD09> { [ hebrew_finalmem,	O	]       };
     key <AD10> { [ hebrew_pe,	P		]	};
 
-    key <AC01> { [ hebrew_shin,	A		]	};
     key <AC02> { [ hebrew_dalet,S		]       };
     key <AC03> { [ hebrew_gimel,D		]       };
     key <AC04> { [ hebrew_kaph,	F		]	};
@@ -52,6 +50,8 @@ xkb_symbols "basic" {
     
     key <TLDE> { [ semicolon,	asciitilde,0x10005b0	]}; // Sheva
     key <AB10> { [ period,	question,  0x10005c3	]}; // Sof Pasuq
+    key <AD03> { [ hebrew_qoph,	E,	EuroSign	]};
+    key <AC01> { [ hebrew_shin,	A,	0x10020AA	]};
   
     // The following may get overriden by the iso9995-3(basic101) symbols.
     // therefore they are included here. 
@@ -77,6 +77,8 @@ xkb_symbols "basic" {
     key <AD12> { [ bracketleft,  braceleft,  0x10005bd	]}; // Meteg
                                       
     key <BKSL> { [ backslash,    bar,  0x10005bb	]}; // Qubuts    
+
+    include "level3(ralt_switch)"
 };
 
 
-- 
1.6.6.1

